packages/terra-clinical-result/src/_ClinicalResultDisplay.jsx

```diff
@@ -78,7 +85,9 @@ const ClinicalResultDisplay = ({
   hasComment,
   conceptDisplay,
   datetimeDisplay,
+  intl,
 }) => {
+  const isValidValue = result?.value;
```

Comment:
How is the incoming `result` prop data look like? Is it the same as the one above? `result?.result?.value` ?

Reply:
Between ClinicalResultDisplay and BloodPressureDisplay they are a little different props wise, you can see how in blood pressure display they are passing in result.result in order to then be able to access the value and unit in the observation file. You can see result.result being passed in here:https://github.com/cerner/terra-clinical/blob/ccf695c432ec48edcab255f6db6c28eb27671ffb/packages/terra-clinical-result/src/_BloodPressureDisplay.jsx#L77

For clinical result display they are just passing in result so result.result.value is not necessary in this case: https://github.com/cerner/terra-clinical/blob/ccf695c432ec48edcab255f6db6c28eb27671ffb/packages/terra-clinical-result/src/_ClinicalResultDisplay.jsx#L122